### PR TITLE
Add 28 more words (incl 2 suggestions) in Jan 2016

### PR DESCRIPTION
**Pull request by qyearsley**
This patch includes a list of words that I've seen misspelled when checking spelling in another project.

### CODE REVIEW

File: data/dictionary.txt [hunk 1]
```diff
@@ -2230,6 +2247,7 @@ intenational->international
 intented->intended, indented,
 intepretation->interpretation
 intepretator->interpretor
+interal->internal
```
Comment (lucasdemarchi) [inline data/dictionary.txt]: I guess this should be a suggestion instead... could be "integral", particularly in SW dealing with math.

File: data/dictionary.txt [hunk 2]
```diff
@@ -2396,6 +2414,7 @@ liuke->like
 livley->lively
 lmits->limits
 loev->love
+loger->logger
```
Comment (lucasdemarchi) [inline data/dictionary.txt]: suggestion, too... It could be "longer"

File: data/dictionary.txt [hunk 3]
```diff
@@ -3875,6 +3903,7 @@ succeded->succeeded
 succeds->succeeds
 succesful->successful
 succesfully->successfully
+succesfully->sucessfully
```
Comment (lucasdemarchi) [inline data/dictionary.txt]: doesn't seem right.

Comment (lucasdemarchi) [inline data/dictionary.txt]: http://dictionary.reference.com/browse/successfully

**Comment by qyearsley** [comment]
Thanks for the review :-) Does it look OK now?

File: data/dictionary.txt [hunk 4]
```diff
@@ -1036,6 +1041,7 @@ constituant->constituent
 constituants->constituents
 constituion->constitution
 constituional->constitutional
+constructable->constructible
```
Comment (lucasdemarchi) [inline data/dictionary.txt]: http://www.collinsdictionary.com/dictionary/english/constructable

it seems to be acceptable, too.

Comment (qyearsley) [inline data/dictionary.txt]: Makes sense; removed this line in the latest update.

**Pull request merged**